pages/[...slug].tsx

```diff
@@ -54,21 +51,42 @@ export default function PostPage({ toc, source, allDocs, nav, frontMatter }) {
 }
 
 export const getStaticProps = async ({ params }) => {
-  const postFilePath = path.join(DOCS_PATH, `${path.join(...params.slug)}.mdx`)
-  const postData = fs.readFileSync(postFilePath)
-  const { content, data } = matter(postData)
+  const [lib] = params.slug
+  const docs = await getDocs(lib, false)
```

Comment:
Leaving the invalidate param as `false` for now until we can find a workaround.

With invalidation enabled:
![image](https://user-images.githubusercontent.com/23324155/143611900-9cef40f7-5e83-467e-b76a-4230f9230f21.png)